frida/__init__.pyi

```diff
@@ -0,0 +1,225 @@
+from typing import Any, Callable, Dict, List, Literal, Optional, Tuple, Union
+
+from . import core
+
+__version__: str
```

Comment:
This doesn't seem to follow PEP-8 wrt. double blank separator lines. Or are the conventions different from .pyi? (I'm new to this 😄)

Reply:
The format is a bit weird but I run it through Black so it should be correct. Anyway I'm a bit conflicted if it's worth merging it if the Python 3 only version would be merged soon. It has type hinting in the code itself which is a lot more pleasant to maintain.

Reply:
Ahh good point. Let's merge the Python 3 only version instead then!